chain-signatures/primitives/src/lib.rs

```diff
@@ -133,18 +134,25 @@ impl Chain {
             Chain::NEAR => "NEAR",
             Chain::Ethereum => "Ethereum",
             Chain::Solana => "Solana",
+            Chain::Hydration => "Hydration",
         }
     }
 
-    pub const fn iter() -> [Chain; 3] {
-        [Chain::NEAR, Chain::Ethereum, Chain::Solana]
+    pub const fn iter() -> [Chain; 4] {
+        [
+            Chain::NEAR,
+            Chain::Ethereum,
+            Chain::Solana,
+            Chain::Hydration,
+        ]
     }
 
     pub fn checkpoint_interval(&self) -> Option<u64> {
         let (key, default) = match self {
             Chain::NEAR => return None,
             Chain::Ethereum => ("CHECKPOINT_INTERVAL_ETHEREUM", 20),
             Chain::Solana => ("CHECKPOINT_INTERVAL_SOLANA", 120),
+            Chain::Hydration => ("CHECKPOINT_INTERVAL_HYDRATION", 120),
```

Comment:
@ChaoticTempest remind me please, why Ethereum has smaller backup interval compared to Solana and now Hydration?